HouseRules_Essentials/README.md

```diff
@@ -11,27 +11,35 @@ HouseRules API.
 
 ### Built-in Rulesets
 
+- __Earth Wind & Fire__ : Not the band. Let's get Elemental.
+- __🕷️Arachnophobia🕷️__ : Offers a fresh adventure to be played on the RootsOfEvil Map.
+  Chased by violent thugs from their ancestral homes in Sunderhaven, the King and Queen flee into the woods.
+  Befriended by money spiders, they hatch a plan to rebuild their fallen empires, but first they're going to need some cash.
+- __🎲LuckyDip🎲__ : Players each start with two 'Drop Chest' cards instead of their normal
+  starting cards, meaning that no two games start the same. Many potions have AOE effect, because it's rude not to share.
+  Many other changes included for faster gameplay with an aim of around 90 minutes per game.
 - __The Swirl__ : Only poison, fireballs and vortexes. Health and POIs aplenty, but must defeat all enemies to escape.
 - __Beat The Clock__ : Ultra health. Ultra card recycling. Only 15 rounds to escape...
 - __Hunter's Paradise__ : Pets, pets, pets! And hunter's mark.
 - __Difficulty Easy__ : Decreased game difficulty for a more casual playstyle.
 - __Difficulty Hard__ : Increased game difficulty for a greater challenge.
 - __Difficulty Legendary__ : Increased game difficulty for those who want to be a legend.
+- __3x3 Potions and Buffs__ : Heal, Strength, Speed, Adamant, Antitoxin, RepairArmor and Bard buffs are 3x3 AOE.
+- __Better Sorcerer__ : 0 Action Cost for Sorcerer's Zap - No other changes. #STS
 - __No Surprises__ :  No surprises in the dark or coming through doors.
 - __Quick and the Dead__ : A mode with a small hand but fast turnaround time on cards means you need to not hesitate.
 
 ### JSON Rulesets
 
 Rulesets may also be configured as JSON files and stored within the game directory `<GAME_DIR>/UserData/HouseRules/<rulesetname>.json`
 
-A selection are available within this repository. These are intended to be fun to play alternative games, and as a good examples for others wanting to create their own rulesets.
+A copy of all built-in rulesets are included as JSON files and, after
+HouseRules installation, can be found in the game directory
```

Comment:
Although this is true for us developers, I didn't think we'd released a set of DLLs where this is the default behaviour yet.

Reply:
With `1.1.0`, the JSON files are bundled in the released ZIP (although they were dropped in there manually during bundling time and not dynamically by the mod).

![image](https://user-images.githubusercontent.com/1426304/160297946-ae4464de-da35-4a4a-a133-223b46c1628b.png)